tests/test_overrides_parser.py

```diff
@@ -1812,3 +1816,352 @@ def test_glob_filter(
 ) -> None:
     strings = ["the", "quick", "brown", "fox", "jumped", "under", "the", "lazy", "dog"]
     assert Glob(include=include, exclude=exclude).filter(strings) == expected
+
+
+# Dummy assignments to allow copy-paste of `TEST_CONFIG_DATA` below from
+# OmegaConf, and properly identify parsing error cases.
+InterpolationSyntaxError = HydraException
+# The OmegaConf exceptions below cannot be identified at parsing time.
+ConfigKeyError = InterpolationTypeError = UnsupportedInterpolationType = None
+
+# DO NOT EDIT `TEST_CONFIG_DATA`!
+# It should be copied regularly from OmegaConf to ensure Hydra's grammar is able
+# to parse similar expressions as OmegaConf's.
+TEST_CONFIG_DATA: List[Tuple[str, Any, Any]] = [
```

Comment:
Don't review the content of `TEST_CONFIG_DATA` in details! It still needs a bit of cleaning up (which will happen first in OmegaConf's related PR, which is where it should be reviewed)